packages/core/src/payment-integration/default-payment-integration-service.ts

```diff
@@ -285,4 +287,15 @@ export default class DefaultPaymentIntegrationService implements PaymentIntegrat
     async validateCheckout(checkout?: Checkout, options?: RequestOptions): Promise<void> {
         await this._checkoutValidator.validate(checkout, options);
     }
+
+    async handlePaymentHumanVerification(
+        errorOrId: Error | string,
+        key?: string,
+    ): Promise<PaymentAdditionalAction> {
+        if (typeof errorOrId === 'string') {
```

Comment:
I checked that we already have the same check inside of `_paymentHumanVerificationHandler.handle` method. Why did you duplicate it here?

Reply:
Without this check I was getting Typescript error here

Reply:
Indeed, i did catch that we have a few declarations of `handle` function. But why we separated the declaration from implementation, i do not know. Anyway thanks for your updates!